lib/zlog.h

```diff
@@ -115,6 +115,12 @@ static inline void zlog_ref(const struct xref_logmsg *xref,
 #define zlog_notice(...) _zlog_ecref(0, LOG_NOTICE, __VA_ARGS__)
 #define zlog_debug(...)  _zlog_ecref(0, LOG_DEBUG, __VA_ARGS__)
 
+#define zlog_debug_if(condition, ...)                                          \
```

Comment:
We have similar (not for logging) `COND_FLAG()` macro, thinking loudly just if we could you the same naming instead? `zlog_debug_cond()`. Another example `bgp_cond_adv_debug()` (same cond naming)

Reply:
I've changed  `zlog_debug_if()` to `zlog_debug_cond()` as suggested.

I think a shorter macro name would be better, given how often it's used and how big an impact it has on line wrapping, but consistency with established convention probably takes precedence.

Reply:
> I think a shorter macro name would be better,

Indeed this is repeating the development of 12272. It started at `zlog_debugif()`, went to `zlog_dbgif()` and finally arrived at `dbg()`